### DISCUSSION TITLE
Option to adjust the brightness/darkness of images rendered by Epub reader

### DISCUSSION THREAD

**Discussion by mozophe** [Kareadita/Kavita#3450] [Ideas]
### Idea Description

Images rendered by the epub reader are not the original images but a darkened version (lowered brightness) to make them easier to read in a dark theme. This darkening of images is hard-coded for all themes (including for white theme). 

https://github.com/Kareadita/Kavita/blob/41c346d5e642cf4596787f9ef09fb7036aef9250/UI/Web/src/app/book-reader/_models/book-white-theme.ts#L119

On a non-dark theme (white and paper themes), darkening of the images makes it difficult to view/read a lot of images properly, especially if the images have text present on it.

An option to adjust this brightness would significantly increase the viewability/readability of the image. Further, darkened images look very dull on the white and paper themes. Users would be able to adjust this brightness if the images included in the epub file are too dark or too bright.

If it takes too much time to develop this feature, a simple fix would be to set the brightness for the white and paper themes as 1, instead of 0.85, to increase the readability of the images, as these two themes are not dark themes. 

Origin of the feature request: https://github.com/Kareadita/Kavita/issues/3449

### Idea Category

User Experience

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
I misread the issue, I'll close this as it is in fact a bug.
